--- system76-scheduler/daemon/Cargo.toml
+++ system76-scheduler/daemon/Cargo.toml
@@ -33,7 +33,8 @@
 # Necessary for deserialization of untagged enums in assignments.
 [dependencies.ron]
 git = "https://github.com/MomoLangenstein/ron"
-branch = "253-untagged-enums"
+#branch = "253-untagged-enums"
+rev = "298f9181d178b6222050433eb8a8be646f75541f"
 
 [dependencies.tracing-subscriber]
 version = "0.3.11"
