--- system76-scheduler/data/assignments.ron
+++ system76-scheduler/data/assignments.ron
@@ -7,6 +7,11 @@
     "pipewire",
     "pipewire-pulse",
     "wireplumber"
+    "Xorg",
+    "gnome-shell",
+    "mutter",
+    "mpv",
+    "vlc",
 ],
 // Very high
 (-9, BestEffort(0)): [
@@ -16,7 +21,6 @@
 // High priority
 (-5, BestEffort(4)): [
     "amsynth",
-    "gnome-shell",
     "kwin",
     "sway",
     "steam",
@@ -24,7 +28,6 @@
     "vrdashboard",
     "vrmonitor",
     "vrserver",
-    "Xorg",
 ],
 // Default
 0: [
@@ -87,6 +90,9 @@
 ],
 // Absolute lowest priority
 (19, Idle): [
+    "nix",
+    "nixos",
+    "nixos-rebuild",
     "apt",
     "apt-get",
     "boinc",